x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/actions_panel/actions_panel.tsx

```diff
@@ -18,18 +18,27 @@ import type { IndexPattern } from '../../../../../../../../src/plugins/data/comm
 import { useDataVisualizerKibana } from '../../../kibana_context';
 import { useUrlState } from '../../../common/util/url_state';
 import { LinkCard } from '../../../common/components/link_card';
+import { ResultLink } from '../../../common/components/results_links';
 
 interface Props {
   indexPattern: IndexPattern;
   searchString?: string | { [key: string]: any };
   searchQueryLanguage?: string;
+  additionalLinks: ResultLink[];
 }
 
 // @todo: Add back create job card in a follow up PR
```

Comment:
We can remove this comment here now that we added the cards back in

Reply:
removed in https://github.com/elastic/kibana/pull/103932/commits/7304780cabd39271a9b99abca627e4ce514ee9cc